web/gds-admin-ui/src/pages/app/details/BasicDetails/components/BusinessInfosForm.js

```diff
@@ -0,0 +1,98 @@
+})
+
+
+function BusinessInfosForm({ data }) {
+    const { register, handleSubmit, formState: { errors, dirtyFields } } = useForm({
+        defaultValues: getBusinessInfosFormInitialValues(data),
+        resolver: yupResolver(schema)
+    })
+    const params = useParams()
+    const dispatch = useDispatch()
+    const safeDispatch = useSafeDispatch(dispatch)
+    const [, setIsOpen] = React.useContext(ModalContext)
+
+    const onSubmit = (data) => {
+        if (params && params.id) {
+            safeDispatch(updateBusinessInfosResponse(params.id, data, setIsOpen))
+        }
+    }
+
+
+    return (
+        <>
+            <h3>Edit Business Information</h3>
+            <p>Please enter extended and basic details about the business entity. Note that the IVMS 101 data contains the legal details and company name information.</p>
+            <Form onSubmit={handleSubmit(onSubmit)}>
+                <Form.Group as={Row} className="mb-2" controlId='website'>
+                    <Form.Label column sm="12" className="fw-normal">
+                        Website
+                    </Form.Label>
+                    <Col sm="12">
+                        <Field.Input isInvalid={!!errors['website']} isValid={dirtyFields['website'] && !errors['website']} type="text" register={register} name="website" />
+                    </Col>
+                    {errors['website'] ? <Form.Control.Feedback type='invalid' className='d-block'>{errors['website'].message}</Form.Control.Feedback> : <Form.Text className='fst-italic'>e.g: https://example.com</Form.Text>}
+                </Form.Group>
+                <Form.Group as={Row} className="mb-2" controlId='established_on'>
+                    <Form.Label column sm="12" className="fw-normal">
+                        Date of Incorporation/Establishment
+                    </Form.Label>
+                    <Col sm="12">
+                        <Field.Input type="date" register={register} name="established_on" />
+                    </Col>
+                </Form.Group>
+                <Form.Group as={Row} className="mb-2" controlId='business_category'>
+                    <Form.Label column sm="12" className="fw-normal">
+                        Business Category
+                    </Form.Label>
+                    <Col sm="12">
+                        <Field.Select register={register} name={`business_category`}>
+                            <BusinessCategory />
+                        </Field.Select>
+                    </Col>
+                    <Form.Text>Please select the entity category that most closely matches your organization.
+                    </Form.Text>
+                </Form.Group>
+                <Form.Group as={Row} className="mb-2" controlId='vasp_categories'>
+                    <Form.Label column sm="12" className="fw-normal">
+                        VASP Category
+                    </Form.Label>
+                    <Col sm="8">
```

Comment:
I think this should be `<Col sm="12">` to match the label and other columns; I'm seeing the following:

![Screen Shot 2022-01-11 at 15 38 59](https://user-images.githubusercontent.com/745966/149025743-3aabda6e-93bf-4316-ad89-d4ad742ddfc2.png)